paddle/gserver/gradientmachines/NeuralNetwork.cpp

```diff
@@ -309,35 +309,35 @@ class CombinedEvaluator : public Evaluator {
   void addEvaluator(std::unique_ptr<Evaluator>&& evaluator) {
     evaluators_.emplace_back(std::move(evaluator));
   }
-  virtual void start() {
+  void start() override {
```

Comment:
OK. I got it from @jacquesqiao 's [comment](https://github.com/PaddlePaddle/Paddle/pull/2591#issuecomment-310841709).